data-esnext.js

```diff
@@ -2653,6 +2653,64 @@ exports.tests = [
       }
     },
   ]
+},
+{
+  name: 'optional catch binding',
+  category: STAGE3,
+  significance: 'tiny',
+  spec: 'https://tc39.github.io/proposal-optional-catch-binding/',
+  subtests: [
+    {
+      name: 'basic',
+      exec: function(){/*
+        try {
+          throw new Error();
+        }
+        catch {
+          return true;
+        }
```

Comment:
I assumed an implicit "return undefined" (false) -- as with the other 2 tests -- but I don't have a strong opinion about it, hence I added it.